reference/extensions.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Extensions
+---
+
+Extensions are similar to [Plugins](../plugins), except for the ways they are not:
+
+  * extensions are typically just modules (really `ActiveSupport::Concern`s) that provide
+  additional logic and behavior to the specific class they're included in.
+    * or, they are simply additional classes wrapped up in a file you `require`.
+  * extensions do not have any configuration data
+  * extensions never instantiated during application loading, nor do they
+  provide callback hooks (beyond that from `ActiveSupport::Concern.included` )
+
+Praxis currently includes the following extensions, all under the `Praxis::Extensions`
+namespace:
+
+  * [`FieldSelection`](./field_selection): adds an enhanced version of `Attributor::FieldSelector`
```

Comment:
Don't think people reading this document will have a clue about what `Attributor::FieldSelector` is. Perhaps write something like: provides a type that allows consumers of the API to select which fields they want in the response.